include/deal.II/fe/mapping_manifold.h

```diff
@@ -259,9 +259,18 @@ class MappingManifold : public Mapping<dim,spacedim>
 
 
     /**
-     * Weights for vertices to use in Manifold::get_new_point().
+     * An array of weights for use in Manifold::get_new_point(). For
```

Comment:
Not technically wrong, but we called `std::array`s arrays elsewhere so we should call this a vector.